castle/src/main/java/io/castle/android/CastleConfiguration.java

```diff
@@ -13,7 +13,7 @@
     private static final int DEFAULT_MAX_QUEUE_LIMIT = 1000;
     private static final boolean DEFAULT_SCREEN_TRACKING_ENABLED = false;
     private static final boolean DEFAULT_USE_CLOUDFLARE_APP = false;
```

Comment:
What is this for?

Reply:
This tells the SDK to use a different api path, see
 https://github.com/castle/castle-android/blob/master/castle/src/main/java/io/castle/android/CastleConfiguration.java#L18